.changeset/tall-numbers-exercise.md

```diff
@@ -0,0 +1,7 @@
+---
+"@hashicorp/design-system-components": patch
+---
+
+`hds-code-editor` - Attaches EditorView instance to the modified element on instantiation
+
+`CodeEditor` - Attaches EditorView instance to the editor element (`.hds-code-editor__editor`) within the Code Editor component
```

Comment:
We aim to use [past tense for changelog entries](https://hashicorp.atlassian.net/wiki/spaces/HDS/pages/3243114706/Changelog+authoring+best+practices#General-guidance)

```suggestion
`hds-code-editor` - Attached `EditorView` instance to the modified element on instantiation

`CodeEditor` - Attached `EditorView` instance to the editor element (`.hds-code-editor__editor`)
```